### PR TITLE
CI Update - Add System.IO.Hashing to .NET 6

### LINKED ISSUE #8090: Missing System.IO.Hashing docs on website
I'm not sure if this is the right repo for this issue, but I was trying to find documentation regarding Crc32, and I noticed this entire library seems to be missing from the docs website from what I can tell.

I was able to find it only on the apisof.net website.
https://apisof.net/catalog/24de7164-bb66-a74a-4765-cf26fb2218a5

### PR DESCRIPTION
**Pull request by gewarren**
Fixes #8090.

### CODE REVIEW

File: xml/System.Net.Sockets/TcpClient.xml [hunk 1]
```diff
@@ -1152,23 +1152,25 @@ The `Available` property is a way to determine whether data is queued for readin
...
+> If you receive a <xref:System.Net.Sockets.SocketException>, use <xref:System.Net.Sockets.SocketException.ErrorCode%2A?displayProperty=nameWithType> to obtain the specific error code. After you've obtained this code, you can refer to the [Windows Sockets version 2 API error code](/windows/desktop/winsock/windows-sockets-error-codes-2) documentation for a detailed description of the error.  
   
 > [!NOTE]
->  This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in the .NET Framework](/dotnet/framework/network-programming/network-tracing).  
+> This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in .NET Framework](/dotnet/framework/network-programming/network-tracing).
```
Comment (bartonjs) [inline xml/System.Net.Sockets/TcpClient.xml:1167]:
```suggestion
> This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in the .NET Framework](/dotnet/framework/network-programming/network-tracing).  

The "the" is part of that other document title.  See https://learn.microsoft.com/en-us/dotnet/framework/network-programming/network-tracing
```

**Review by bartonjs — COMMENTED** [review]
Not sure why the structural addition of System.IO.Hashing is intermixed with some content cleanup in System.Net; but other than one change I disagree with to the System.Net documentation (I believe "match the title of the destination page of a link" wins out over house-style), LGTM.

File: xml/System.Net.Sockets/TcpClient.xml [hunk 2]
```diff
@@ -1152,23 +1152,25 @@ The `Available` property is a way to determine whether data is queued for readin
...
   
 > [!NOTE]
->  This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in the .NET Framework](/dotnet/framework/network-programming/network-tracing).  
+> This member outputs trace information when you enable network tracing in your application. For more information, see [Network Tracing in .NET Framework](/dotnet/framework/network-programming/network-tracing).  
   
 > [!NOTE]
-> If you receive NotSupportedException with message `This protocol version is not supported` while using IPv6 address, then make sure you enabled IPv6 in constructor by passing <xref:System.Net.Sockets.AddressFamily.InterNetworkV6>.
+> If you receive a `NotSupportedException` with the message `This protocol version is not supported` while using IPv6 address, make sure you enabled IPv6 in the constructor by passing <xref:System.Net.Sockets.AddressFamily.InterNetworkV6>.
```
Comment (gewarren) [inline xml/System.Net.Sockets/TcpClient.xml:1170]:
```suggestion
> If you receive a `NotSupportedException` with the message "This protocol version is not supported" while using IPv6 address, make sure you enabled IPv6 in the constructor by passing <xref:System.Net.Sockets.AddressFamily.InterNetworkV6>.
```

**Review by carlossanlop — APPROVED** [review]

**Comment by gewarren** [comment]
> Not sure why the structural addition of System.IO.Hashing is intermixed with some content cleanup in System.Net; but other than one change I disagree with to the System.Net documentation (I believe "match the title of the destination page of a link" wins out over house-style), LGTM.

The CI job runs over all the assemblies, not just the new one. And I added some manual touch-ups to a file that was changed anyway since there were some bad grammar errors.

**Pull request merged**